tests/core/utils/test_block.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+
+import rlp
+
+from evm.utils.blocks import (
+    add_block_number_to_hash_lookup,
+    get_score,
+    persist_block_to_db,
+    get_block_header_by_hash,
+    lookup_block_hash,
+)
+
+from evm.db import (
+    get_db_backend,
+)
+
+from evm.vm.flavors.frontier.blocks import (
+    FrontierBlock,
+)
+
+from evm.rlp.headers import (
+    BlockHeader,
+)
+
+from evm.utils.db import (
+    make_block_hash_to_score_lookup_key,
+    make_block_number_to_hash_lookup_key,
+)
+
+db = get_db_backend()
```

Comment:
Same approach recommended here for these three globals.  I'd make them pytest fixtures.

```python
@pytest.fixture
def db():
    return get_db_backend()

@pytest.fixture
def header():
    return BlockHeader(...)

@pytest.fixture
def block(header, db):
    return FrontierBlock(header, db)

# Then in your tests
def test_something(db, block):
    ...
```

This ensures proper isolation between tests.